app/controllers/booths_controller.rb

```diff
@@ -0,0 +1,52 @@
+class BoothsController < ApplicationController
+  before_action :authenticate_user!
+  load_resource :conference, find_by: :short_title
+  load_and_authorize_resource through: :conference
+
+  def index; end
```

Comment:
Even if I am an admin/organizer, in the non-admin view I only want to see my own booths (ie booths I am the submitter/responsible of)